### ISSUE TITLE
Default milestone for the issues raised by 0pdd

### ISSUE THREAD

**Issue by dgroup** [yegor256/0pdd#258]
Are there any features which allow setting the default milestone for each raised task by 0pdd?

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@dgroup[/z](https://www.0crat.com/u/dgroup) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3T46CUJJ) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@dgroup not yet, but we can add it. Maybe also default assignee, default label(s), default project(s)?

**Comment by dgroup** [comment]
@yegor256, regarding default
1. `assignee(s)`
     Agree
2. `label(s)` 
    I think its already done.
3. `project(s)`
   Sorry, but could you please clarify this point?

**Comment by dgroup** [comment]
@yegor256, yes, I've checked out the "project" functionality for GitHub issues, and you are right, that would be good to have the default project.

For example, if only 1 project has status open than it can be used by default.
Otherwise, we need to specify the project ID/name in `0pdd.yml` explicitly.

**Comment by frou** [comment]
The mentioned ability to configure a default issue assignee would make a lot of sense for when a single developer works alone on a project but still wants to use PDD.

**Comment by paulodamaso** [comment]
@dgroup WDYT about if no default milestone is defined, copy the parent issue milestone?

**Comment by dgroup** [comment]
@paulodamaso in case if issue **A** has a parent **B** (with milestone) it means that **B** can't be delivered without **A**, thus issue **A** should be delivered within the same milestone.

So, yes, copy the parent issue milestone.

**Comment by paulodamaso** [comment]
@dgroup Yes, it should be explictly added to issue **A** when pdd creates it
